src/js/components/Paragraph.js

```diff
@@ -31,5 +32,6 @@ Paragraph.propTypes = {
   align: PropTypes.oneOf(['start', 'center', 'end']),
   id: PropTypes.string,
   margin: PropTypes.oneOf(['none', 'small', 'medium', 'large']),
-  size: PropTypes.oneOf(['small', 'medium', 'large'])
+  size: PropTypes.oneOf(['small', 'medium', 'large']),
+  width: PropTypes.oneOf(['small', 'medium', 'large'])
```

Comment:
Would it work to key the width and max-width off of the 'size' property value? I'm thinking that the max readable text width is relative to the font size, so the larger the font size, the larger the width and max-width.